### DISCUSSION TITLE
Where are the settings stored?

### DISCUSSION THREAD

**Discussion by KodeXall** [jonaskohl/CapsLockIndicator#123] [Q&A] [answered]
Is there a way to export / save user's configuration?

They only created on drive C was a LOG located in

**C:\Users\YOUR-USER-NAME\AppData\Local\Microsoft\CLR_v4.0_32\UsageLogs**

And in the Registry of Windows 10 I have not found anything substantial

**Comment by jonaskohl** [comment]
The settings are stored in a settings file. This file can be copied to other machines to export/import the settings. For more information about the settings file and its path, please refer to [this knowledgebase article](https://cli.jonaskohl.de/kb/?a=settingsfile).

**Comment by KodeXall** [comment, marked as answer]
Yes, the config file is available at

**C:\Users\YOUR-USER-NAME\AppData\Roaming\Jonas Kohl\CapsLock Indicator\settings\any**

I was not able to find that folder with search feature [of e.g. a file-manager like FreeCommander], because it is named

**CapsLock Indicator**

while the software is called

**CapsLockIndicator**

i.e. without a space. And thus this should be changed, so that the next new user will not waste time on such trivial matter

**Comment by jonaskohl** [comment]
The software is actually called "CapsLock Indicator" **with** a space. Also, as the settings path is documented well enough in addition to keeping backwards compatiblity, the settings path will be kept as-is.
